src/V3EmitCHeaders.cpp

```diff
@@ -254,8 +263,141 @@ class EmitCHeader final : public EmitCConstInit {
         puts("return !(*this == rhs);\n}\n");
         puts("};\n");
     }
+
+    // getfunc: VL_ASSIGNSEL_XX(rbits, obits, off, lhsdata, rhsdata);
+    // !getfunc: VL_ASSIGNSEL_XX(rbits, obits, lhsdata, rhsdata, off);
+    void emitVlAssign(const AstNodeDType* const lhstype, const AstNodeDType* rhstype,
+                      const std::string& off, const std::string& lhsdata,
+                      const std::string& rhsdata, bool getfunc) {
+        puts("VL_ASSIGNSEL_");
+        puts(lhstype->charIQWN());
+        puts(rhstype->charIQWN());
+        // Put lhs width
+        puts("(" + std::to_string(lhstype->width()) + ", ");
```

Comment:
```suggestion
        puts("(" + std::to_string(lhstype->width()) + ", ");  // LHS width
```

Similar later lines - having it denser helps readability so function fits on one screen

Reply:
Fixed